app/js/lib/view_handler/display_bank_data.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+	_ = require( 'lodash' ),
+	BankDataDisplayHandler = {
+		accountNumber: null,
+		bankCode: null,
+		bankName: null,
+		update: function ( formContent ) {
+			if ( formContent.debitType === 'sepa' ) {
```

Comment:
Oh, this is just the pre-confirmation display. Please ignore my comment above...